docs/content/docs/build/experimental-features.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Experimental Features
+aliases:
+  - "/docs/build/experimental-features/"
+author: scottn12
+---
+
+## Overview
+
+At any given time, there are experimental features within FluidFramework that are still being actively developed/tested. In order to allow developers to adopt and test these features in a controlled environment, we introduced experimental feature gates to enable these features and easily disable them if they cause unexpected behavior.
+
```

Comment:
I recommend that you add a brief definition of "gate" here and how gates are implemented.